test/specs/errors.specs.ts

```diff
@@ -33,4 +33,51 @@ describe("Errors", () => {
             "Error: Code: REQUEST_ERROR, HttpCode: 400, Errors: REQUEST_ERROR, REQUEST_ERROR (my no reason field)"
         );
     });
+
+    it("should serialize the ResponseError correctly", async () => {
```

Comment:
This description is duplicated. 🤔

Reply:
Ups. Seems that we already had the test. Thanks 🙂